docs/introduction/architecture.md

```diff
@@ -3,51 +3,51 @@ title: Shard Architecture
 
 ## Consensus
 
-Calimero is using the NEAR consensus algorithm called Nightshade. It is modeled as a single blockchain, in which each block logically contains all the transactions for all the shards, and changes the whole state of all the shards. Physically, however, no participant downloads the full state or the full logical block. Instead, each participant of the network only maintains the state that corresponds to the shards that they validate transactions for, and the list of all the transactions in the block is split into physical chunks, one chunk per shard. You can find more information about how it works on on this [NEAR paper](https://near.org/papers/nightshade/#nightshade).
+Calimero utilizes the Nightshade consensus algorithm, which is based on the NEAR consensus algorithm. Nightshade models the blockchain as a single entity where each block logically contains all transactions for all the shards, thus changing the state of all the shards. However, participants in the network do not download the full state or logical block physically. Instead, each participant maintains only the state corresponding to the shards they validate transactions for. The list of transactions in a block is divided into physical chunks, with one chunk per shard. For more detailed information on how it works, refer to the [NEAR paper](https://near.org/papers/nightshade/#nightshade).
 
 ## Proof of Authority Shard
 
-In a proof of authority (PoA) shard, all validators have same voting power and number of blocks. PoA shards are often used in private or permissioned blockchain networks, where the validators are known and trusted entities, such as consortium members or enterprise partners. PoA shards offer faster transaction speeds and lower energy consumption compared to proof of work (PoW) or proof of stake (PoS) consensus mechanisms.
+In a proof of authority (PoA) shard, all validators have equal voting power and an equal number of blocks. PoA shards are commonly used in private or permissioned blockchain networks where validators are known and trusted entities, such as consortium members or enterprise partners. PoA shards offer faster transaction speeds and lower energy consumption compared to proof of work (PoW) or proof of stake (PoS) consensus mechanisms.
 
 ## Proof of Stake Shard
 
 The proof of stake (PoS) shard, provides users the ability to have different amount of voting power/blocks produced in the network. PoS Network this can be customised based on the preferences of the network owners.Validators are required to lock up a certain amount of cryptocurrency or token as collateral, which incentivizes them to act honestly and secure the network. PoS shards are designed to be more energy-efficient, secure, and decentralized than PoW shards, as they do not require large amounts of computational power and electricity to perform the validation process. 
```

Comment:
Should network in PoS Network be capitalized

Reply:
There is a space missing before Validators

Reply:
as a collateral

Reply:
to act honestly.

Reply:
> as a collateral

"as collateral" is more commonly used and generally preferred.

Reply:
> to act honestly.



> to act honestly.

I think the full sentence is okay as well,  no need to add the stop`.`  at honestly